autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -20,6 +32,15 @@ const CourseSelectColumn: React.FC<RouteComponentProps> = () => {
 
   const rows: JSX.Element[] = [];
 
+  // Recalculate padding of course cards after rendering component or resizing window
+  React.useEffect(() => {
```

Comment:
Could this be simplified to `React.useEffect(updateCourseCardPadding, [rows]);`?

Reply:
Yes, I changed it to that now